frontend/src/utils/index.js

```diff
@@ -39,6 +39,8 @@ export const getRelativeResultPathName = (project = {}, result = {}) =>
 export const getGrandParentDirectoryName = (result = {}) =>
   path.basename(path.resolve(result.pathName, '..'));
 
+export const getResultGroupName = (result = {}) => getGrandParentDirectoryName(result);
```

Comment:
Is this method an alias?

Reply:
This method is just an alias. I'll rename the function and remove the alias.